NitroxClient/MonoBehaviours/Chat.cs

```diff
@@ -27,7 +27,7 @@ public void OnConsoleCommand_chat(NotificationCenter.Notification n)
                 }
 
                 Multiplayer.PacketSender.SendChatMessage(text);
-                ErrorMessage.AddMessage("Me: " + text);
+                ErrorMessage.AddMessage("Me: " + text); // Assumption: This method will only be called in-game
```

Comment:
What does this comment mean?  This is a monobehaviour that should only be called in the game.